.env.dev

```diff
@@ -15,7 +15,7 @@ TWILIO_AUTH_TOKEN=
 TWILIO_NUMBER=
```

Comment:
I think by not having to have users copy `.env.dev.example` -> `.env.dev` we can avoid the case where a new change in `.env.dev.example` doesn't get reflected in their `.env.dev` or `.env` because they forgot to copy any new additions.

The docker containers are started w/ 3 env files:
- `.env.dev`
- `.env.${DB}.dev`
- `.env` (gitignored; variables set in  this file get the highest priority)

Reply:
I propose to keep the `.env` untouched for dev environment, since `.env` is used for `docker-compose.yml` by default. Would be great to keep them separate, and use only `.env.dev` and `.env.${DB}.dev` for dev environment.

Reply:
good point 👍